components/StyledModal.js

```diff
@@ -118,7 +118,7 @@ const CloseIcon = styled(Times)`
 export const ModalHeader = ({ children, onClose, hideCloseIcon, ...props }) => (
   <Header {...props}>
     {children || <div />}
-    {!hideCloseIcon && <CloseIcon onClick={onClose} />}
+    {!hideCloseIcon && <CloseIcon className="closeIcon" onClick={onClose} />}
```

Comment:
Using `className` with styled-components is an anti-pattern. If this really is needed (ideally, we should find a way to work with the existing layout), then you can export the `CloseIcon` component and use it like:

```es6
const test = styled.div`
  ${CloseIcon} {
    /* put styles here */
  }
`
```

Reply:
@Betree : Thanks. I've used your suggestion and that works fine. When you say "we should find a way to work with the existing layout" do you mean to say that the design should be adapted or that we should just use the existing close icon style? 🤔

Reply:
Well, it seemed strange to me that we have to add these special styles for the close icon here while we didn't have to do that for other modals that contain an image in the header, like the Add funds one:
![image](https://user-images.githubusercontent.com/1556356/125042417-15303b80-e09a-11eb-9390-dc58c3284d91.png)

I think that's something we should be consistent with and update more globally (centering the :x: vertically in the modal header) but we can keep that for later, that's not a blocker here.

Reply:
Thanks the for idea. Let me open a issue for this soon. 😄

Reply:
Done at, https://github.com/opencollective/opencollective/issues/4440.